docs/core/examples/overview.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+Prefect includes a number of examples covering different features. These can be
+viewed live in the docs, or accessed from the GitHub repo
+[here](https://github.com/PrefectHQ/prefect/tree/master/examples).
+
+## Running with Prefect Cloud or Server
+
+When running with Prefect Cloud or Prefect Server, you can register all the
+examples in a new project by running the following:
```

Comment:
It'd be nice to note somewhere in this file that each example comes with a command to register it alone. It'd be hesitant to whole-sale import like this.